samples/torch/src/main/kotlin/HelloTorch.kt

```diff
@@ -0,0 +1,624 @@
+import kotlinx.cinterop.*
+import torch.*
+import platform.posix.*
```

Comment:
Which POSIX APIs are used here?

Reply:
Methods for reading files, only loading the digit data (see MNIST class from line 517)